crates/primitives/src/bits/macros.rs

```diff
@@ -684,6 +684,81 @@ macro_rules! bytes {
     }};
 }
 
+/// Implements the `ssz::Decode` and `ssz::Encode` traits for a given type
+/// with a specified fixed length.
+///
+/// This macro is designed to be used with types that have a known, fixed size
+/// when encoded or decoded using the Simple Serialize (SSZ) encoding scheme.
+/// It generates implementations of the `ssz::Decode` and `ssz::Encode` traits
+/// which enforce that the SSZ-encoded byte representation of the type has a
+/// length exactly equal to the specified `fixed_len`.
+///
+/// The `ssz::Decode::from_ssz_bytes` function provided by this implementation
+/// will produce an error if the length of the byte slice provided does not
+/// match `fixed_len`. Conversely, the `ssz::Encode::ssz_append` function will
+/// append exactly `fixed_len` bytes to the provided buffer.
+#[macro_export]
+#[cfg(feature = "ssz")]
+macro_rules! impl_ssz_fixed_len {
```

Comment:
This shouldn't be exported and should be `#[doc(hidden)]`. Should follow the examples above like `impl_arbitrary` and add it to `wrap_fixed_bytes`